packages/shared/src/index.ts

```diff
@@ -1,13 +1,10 @@
 export * from "./constants";
-export * from "./queries";
 export * from "./interfaces/filters";
 export * from "./interfaces/orderBy";
 export * from "./interfaces/cloudConfigSchema";
 export * from "./interfaces/parseDbOrg";
 export * from "./tableDefinitions";
 export * from "./types";
-export * from "./filterToPrisma";
-export * from "./orderByToPrisma";
```

Comment:
Those are used on the backend only and use the logger module. If we keep them at this level, some *.tsx files try to import them which breaks the compilation. Hence, I moved them into the `server` sub-module.